src/main/java/DukeException.java

```diff
@@ -0,0 +1,7 @@
+public class DukeException extends Exception {
+    protected String errorMsg;
```

Comment:
I think "errorMessage" would be better than "errorMsg" here as we are required to avoid 'texting-style' spelling.

Reply:
thank you